requirements.txt

```
@@ -0,0 +1 @@
+yamllint
```

let's pin the version while we're at it
```suggestion
yamllint==1.26.3
```